src/components/safe-apps/SafeAppsSDKLink/SafeAppsSDKLink.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react'
+import Link from 'next/link'
+import Box from '@mui/material/Box'
+import Collapse from '@mui/material/Collapse'
+import Typography from '@mui/material/Typography'
+import MUILink from '@mui/material/Link'
+import Fab from '@mui/material/Fab'
+import KeyboardDoubleArrowUpRoundedIcon from '@mui/icons-material/KeyboardDoubleArrowUpRounded'
+
+import CodeIcon from '@/public/images/apps/code-icon.svg'
+import css from './styles.module.css'
+
+const SafeAppsSDKLink = () => {
+  const [openModal, setOpenModal] = useState<boolean>(false)
+
+  const safeAppsSDKUrl = 'https://docs.gnosis-safe.io/learn/safe-tools/sdks/safe-apps'
```

Comment:
Would it be better to have this as a field in the config service? At this stage, we already will require one deployment in the future to change the link to a new domain + with the addition of Tomiwa docs are expected to improve and therefore change a lot

Reply:
Maybe too much to create a dynamic field in the config service for something that shouldn't change that often.

In this case it's true we will need to update it once the new domain and docs are ready but then we shouldn't

What I would definitely do is to move this to a constant in this file, so it's easier to track
https://github.com/safe-global/web-core/blob/dev/src/config/constants.ts